pyfar/classes/transmission_matrix.py

```diff
@@ -132,21 +133,34 @@ def from_tmatrix(cls, data, frequencies, comment = ""):
         """
         return cls(data, frequencies, comment)
 
+    def _tmat_from_abcd(A, B, C, D):
+        """Create a T-matrix from A-, B-, C-, D-data."""
```

Comment:
Maybe something like?
```suggestion
    def _tmat_from_scalar_abcd(A, B, C, D):
        """Create a array representing a frequeny-independent T-matrix from A-, B-, C-, D-data."""
```